project.clj

```diff
@@ -126,7 +126,7 @@
              :ezbake {:dependencies ^:replace [[puppetlabs/puppetdb ~pdb-version]
                                                [org.clojure/tools.nrepl "0.2.3"]]
                       :name "puppetdb"
-                      :plugins [[puppetlabs/lein-ezbake "0.2.9"
+                      :plugins [[puppetlabs/lein-ezbake "0.3.8-SNAPSHOT"
```

Comment:
This won't work once merged, we only currently work against a released version, unless we keep the development workflow in place or something. More likely we'll get a release of 0.3.8 soon.